README.rst

```diff
@@ -25,54 +25,56 @@ nocaselist - A case-insensitive list for Python
 Overview
 --------
 
-TBD
+Class ``NocaseList`` is a case-insensitive list that preserves
+the lexical case of its items.
+
+Example:
+
+.. code-block:: bash
+
+    $ python
+    >>> from nocaselist import NocaseList
+
+    >>> list1 = NocaseList(['Alpha', 'Beta'])
+
+    >>> print(list1)  # Any access is case-preserving
+    ['Alpha', 'Beta']
+
+    >>> 'ALPHA' in list1  # Any lookup or comparison is case-insensitive
+    True
+
 
 Installation
 ------------
 
-To install the latest released version of the nocaselist
-package into your active Python environment:
+To install the latest released version of the nocaselist package into your
+active Python environment:
 
 .. code-block:: bash
 
     $ pip install nocaselist
 
-This will also install any prerequisite Python packages.
+This would also install any prerequisite Python packages, but the nocaselist
```

Comment:
Why not just say that it has no prerequisites.

Reply:
DONE